docs/diracdoctools/cmd/concatcfg.py

```diff
@@ -11,8 +11,11 @@
 from diracdoctools.Config import Configuration
 from diracdoctools.Utilities import makeLogger
 
-from DIRAC.Core.Utilities.CFG import CFG
-from DIRAC import S_OK, S_ERROR
+try:
```

Comment:
Maybe you just add a comment here that this try/except was added for python 3 compatibility

Reply:
Yes, very good point. I added a comment.